example/package.json

```diff
@@ -14,31 +14,32 @@
     "postinstall": "patch-package && expo prebuild"
   },
   "dependencies": {
-    "@expo/metro-config": "~0.18.1",
-    "@expo/metro-runtime": "^3.2.1",
+    "@expo/metro-config": "~0.18.11",
+    "@expo/metro-runtime": "~3.2.3",
     "@react-native-async-storage/async-storage": "^1.23.1",
     "@react-native-community/slider": "^4.5.2",
     "@react-native-community/viewpager": "^5.0.11",
-    "@react-native-masked-view/masked-view": "^0.2.8",
+    "@react-native-masked-view/masked-view": "0.3.1",
     "@react-navigation/elements": "^1.3.7",
     "@react-navigation/native": "^6.1.17",
     "@react-navigation/stack": "^6.3.29",
     "@swmansion/icons": "^0.0.1",
-    "expo": "^51.0.0",
-    "expo-camera": "~15.0.9",
+    "expo": "~51.0.38",
+    "expo-build-properties": "^0.12.5",
+    "expo-camera": "~15.0.16",
     "expo-font": "~12.0.10",
     "hoist-non-react-statics": "^3.3.2",
     "invariant": "^2.2.4",
     "patch-package": "^6.5.1",
     "postinstall-postinstall": "^2.1.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "react-native": "0.74.1",
+    "react-native": "0.75.4",
```

Comment:
Why not 0.76?

Reply:
For now expo migrating tools for `rn 0.75` -> `rn 0.76` and `sdk 51` -> `sdk 52` are not yet supported, and I could not manually get it working by bumping everything's version to latest.
- `expo install expo@^52.0.0 --fix` < crashes upon usage
- `expo-doctor` < doesn't support `sdk 52` yet
- `expo upgrade helper` < doesn't support `sdk 52` yet

I believe it'd be best to wait for one of these tools to be supported. It could take anywhere from a couple of days to a couple of weeks, so i think it makes the most sense to keep it at `0.75` for now, and open a separate PR bumping the version to `0.76` once `expo` migrating tools will start working.